components/tabs/demo/tab-custom.js

```diff
@@ -0,0 +1,38 @@
+import { css, html, LitElement, unsafeCSS } from 'lit';
+import { getFocusPseudoClass } from '../../../helpers/focus.js';
+import { TabMixin } from '../tab-mixin.js';
+
+class TabCustom extends TabMixin(LitElement) {
+
+	static get styles() {
+		const styles = [ css`
+			.d2l-tab-custom-content {
+				margin: 0.5rem;
+				overflow: hidden;
+				padding: 0.1rem;
+				white-space: nowrap;
+			}
+			:host(:first-child) .d2l-tab-custom-content {
+				margin-inline-start: 0;
+			}
+			:host(:${unsafeCSS(getFocusPseudoClass())}) > .d2l-tab-custom-content {
+				border-radius: 0.3rem;
+				box-shadow: 0 0 0 2px var(--d2l-color-celestine);
```

Comment:
Should we be doing something different from `box-shadow`, given the stuff Dave recently worked on for high contrast? A little worried people might copy this if it's not what we want.